Source/Common/UICollectionView+IGListManagementPerformable.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "IGListManagementPerformable.h"
+
+@interface UICollectionView (IGListManagementPerformable) <IGListManagementPerformable>
+@end
```

Comment:
Only diffing stuff Should be in `Common`, right? I think this needs to move to `Source/`